packages/yourloops/components/icons/RoundedHospitalIcon.tsx

```diff
@@ -1,6 +1,6 @@
 /**
- * Copyright (c) 2021, Diabeloop
```

Comment:
hummm it should be 2021-2022 as this file existed in 2021 :slightly_smiling_face:

Reply:
This one is a new one ! I added it with this PR